metron-analytics/metron-profiler-client/src/test/java/org/apache/metron/profiler/client/ProfileWriter.java

```diff
@@ -95,4 +102,22 @@ private void write(ProfileMeasurement m, List<Object> groups) {
     hbaseClient.mutate();
   }
 
+  public static void main(String[] args) throws Exception {
+    RowKeyBuilder rowKeyBuilder = new SaltyRowKeyBuilder();
+    ColumnBuilder columnBuilder = new ValueOnlyColumnBuilder();
+
+    Configuration config = HBaseConfiguration.create();
+    config.set("hbase.master.hostname", "node1");
+    config.set("hbase.regionserver.hostname", "node1");
+    config.set("hbase.zookeeper.quorum", "node1");
+
+    HTableProvider provider = new HTableProvider();
+    HTableInterface table = provider.getTable(config, "profiler");
+
+    long when = System.currentTimeMillis() - TimeUnit.DAYS.toMillis(2);
+    ProfileMeasurement measure = new ProfileMeasurement("profile1", "192.168.66.121", when, 15, TimeUnit.MINUTES);
```

Comment:
Running tests with interval set to 15 minutes will make unit testing and integration testing unreasonably long.  Please use small numbers of seconds as duration setting for all tests.